assignments/array-methods.js

```diff
@@ -80,17 +80,27 @@ console.log(allCaps);
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.
-// Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result.
+// Get a list of runners with large sized shirts so they can choose a different size.
+// Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result.
 
 console.log('CHALLENGE 3 👇');
 
 let largeShirts = [];
+
+largeShirts = runners.filter(runner => {
+    if(runner.shirt_size === "L") {
+        return runner;
+    }
+});
```

Comment:
Good job on using filter here. A better way to shorten this one though is to declare largeShirts in the same line as filter and then return the shirts without the if statement, but there are different ways to go about this and you did a good job either way!